src/Kobo/SystemDialog.cpp

```diff
@@ -63,7 +67,10 @@ class SystemWidget final
   void ExportUSBStorage();
   void IncreaseBacklightBrightness();
   void DecreaseBacklightBrightness();
-  void UpdateBacklightButtons(int percent);
+  void UpdateBacklightBrightnessButtons(int percent);
```

Comment:
What does the parameter mean? Prior to the rename, it was clear, but it's not any longer.

Reply:
The backlight has two controls: brightness and colour.  The existing code ignored colour and referred to the backlight brightness as "...Backlight..." in function names  The change to "...BacklightBrightness..." makes it easier to add other backlight controls (like colour) and actually makes the original parameter meaning clearer!